spec/components/post_destroyer_spec.rb

```diff
@@ -676,7 +676,15 @@
       second_post.expects(:update_flagged_posts_count)
       PostAction.defer_flags!(second_post, moderator)
       second_post.reload
-      PostDestroyer.new(moderator, second_post).destroy
+      PostDestroyer.new(moderator, second_post, agree_flags: true).destroy
+      expect(
+        Topic.where(title: I18n.t('system_messages.flags_agreed_and_post_deleted.subject_template')).exists?
+      ).to eq(false)
```

Comment:
This spec is no longer accurate because we queue jobs by default now. Since the job is queued and never executed, the topic is never created. We should update this to check for the jobs in the queue.